dashboard/src/utility/dashboardClasses.js

```diff
@@ -0,0 +1,9 @@
+import classNames from 'classnames'
+import snakeCase from 'lodash/snakecase'
+
+const dashboardClasses = (owner, ...args) => {
```

Comment:
1. :+1:
2. :+1: to `component-OriginalName`; that increases searchability a lot
3. :+1: